ci_dependencies.sh

```diff
@@ -859,3 +848,13 @@ fi
 fi
 
 
+# Start of recipe for dependency: systemd
```

Comment:
I got what is nagging me: why is this change not reflected in `.travis.yml`? Did you `ProjectXML -A` to rewrite and carefully `git difftool` that (and other "semistatic") file in as automated and comfy manner as possible?

Reply:
Should be fixed by applying https://github.com/aquette/fty-security-wallet/pull/1